Corona-Warn-App/src/main/res/values-de/legal_strings.xml

```diff
@@ -31,13 +31,15 @@
     <string name="submission_your_consent_agreement_share_symptoms">"<b>Im Falle von Schnelltests werden nur Nutzer der Corona-Warn-App gewarnt. Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese Daten geteilt.</b>\n\nSie können Ihr Einverständnis zurücknehmen, indem Sie oben „Andere warnen“ deaktivieren."</string>
 
     <!-- YTXT: Body for your consent screen agreement share symptoms with additional hint for test result-->
-    <string name="submission_your_consent_agreement_share_symptoms_2"><b>Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese geteilt.</b>\n\nSie können Ihr Einverständnis zurücknehmen, indem Sie oben „Andere warnen“ deaktivieren. Ihr Testergebnis wird Ihnen anschließend angezeigt.</string>
+    <string name="submission_your_consent_agreement_share_symptoms_2"><b>Im Falle von Schnelltests werden nur Nutzer der Corona-Warn-App gewarnt. Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese Daten geteilt.</b>\n\nSie können Ihr Einverständnis zurücknehmen, indem Sie oben „Andere warnen“ deaktivieren. Ihr Testergebnis wird Ihnen anschließend angezeigt.</string>
     <!-- YTXT: Body for keys submission no consent text first part-->
     <string name="submission_no_consent_your_consent_subsection_body_first_part">"Durch Antippen von „Einverstanden“ willigen Sie wie folgt ein:"</string>
     <!-- YTXT: Body for keys submission no consent text second part-->
-    <string name="submission_no_consent_your_consent_subsection_body_second_part">"<b>Die App teilt Ihr Testergebnis, um Nutzer von offiziellen Corona-Apps, denen Sie begegnet sind, zu warnen. Die Warnung funktioniert in den oben genannten Ländern. Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese geteilt.</b> Bitte geben Sie Ihr Testergebnis (genauer gesagt: Ihre Zufalls-IDs der letzten 14 Tage) im nächsten Schritt frei, indem Sie „Teilen“ antippen."</string>
+    <string name="submission_no_consent_your_consent_subsection_body_second_part">"<b>Die App teilt Ihr Testergebnis, um Nutzer, denen Sie begegnet sind, zu warnen. Dies betrifft Nutzer von Corona-Apps der oben genannten Länder und Nutzer, die zeitgleich am selben Event oder Ort wie Sie eingecheckt waren. Im Falle von Schnelltests werden nur Nutzer der Corona-Warn-App gewarnt. Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese Daten geteilt.</b>"</string>
```

Comment:
It took me a while to understand

> Dies betrifft Nutzer von Corona-Apps der oben genannten Länder und Nutzer, die zeitgleich am selben Event oder Ort wie Sie eingecheckt waren.

that these are two separate conditions. In the first readings I understood that **users of foreign contries who were also checked in** with event registration will be warned.

I'd suggest to make two sentences out of it, or add "...und außerdem":
"Dies betrifft Nutzer von Corona-Apps der oben genannten Länder **und außerdem** Nutzer, die zeitgleich am selben Event oder Ort wie Sie eingecheckt waren."

Also I stumbled now upon missing "gendering" -> Nutzer:innen etc... I don't know what are your plans for this issue, but I guess all German documents/strings would need to be inspected?

Reply:
Hi, thanks for your feedback. I'll forward both suggestions to our User Assistent colleagues. Won't be changed in this PR as these text are set since a while.